python/MooseDocs/extensions/bibtex.py

```diff
@@ -85,8 +85,8 @@ def preExecute(self, content):
 
     def postTokenize(self, ast, page, meta, reader):
         if self.__citations:
-            meta.getData('citations').update(self.__citations)
-            self.__citations.clear()
+            meta.getData('citations').extend(self.__citations)
+            del self.__citations[:]
```

Comment:
```suggestion
            del self.__citations[:] # TODO: In python 3 this should be a .clear()
```

Reply:
Just a reminder for later this year when we update

Reply:
Makes sense.  I've re-pushed.